plugins/nf-validation/src/main/nextflow/validation/FormatValidators/PathValidator.groovy

```diff
@@ -0,0 +1,15 @@
+package nextflow.validation
+
+import java.nio.file.Path
+
+import org.everit.json.schema.FormatValidator
+import nextflow.Nextflow
+
+public class PathValidator implements FormatValidator {
```

Comment:
What is this validating? Isn't it be the same as `PathExistsValidator`?

Reply:
This mainly checks if a `Path` object can be created from the input (the format is mainly used in the conversion later on, but the parser gives an error if it's not a valid format)